src/components/scene/vr-mode-ui.js

```diff
@@ -71,6 +45,8 @@ module.exports.Component = registerComponent('vr-mode-ui', {
 
     this.orientationModalEl = createOrientationModal(this.exitVR);
     this.el.appendChild(this.orientationModalEl);
+
+    this.attachEventListeners();
```

Comment:
@ngokevin: https://github.com/aframevr/aframe/pull/1236/files#diff-2e99cd43fe5dd47f035f54a4fcc06f44R77 is the line I care about. I want to call that _after_ update. I'm moving things around to make this happen.